From: Ben Hutchings <ben@decadent.org.uk>
Subject: AUFS_PROC_MAP is BROKEN
Date: Thu, 07 Feb 2013 05:02:18 +0000
Bug-Debian: http://bugs.debian.org/693332

This config option depends on aufs3-proc_map.patch, which we don't apply.

--- a/fs/aufs/Kconfig
+++ b/fs/aufs/Kconfig
@@ -102,6 +102,7 @@ config AUFS_RDU
 
 config AUFS_PROC_MAP
 	bool "support for /proc/maps and lsof(1)"
+	depends on BROKEN
 	depends on PROC_FS
 	help
 	When you issue mmap(2) in aufs, it is actually a direct mmap(2)
